app/src/main/res/drawable/bg_rain_circle.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="oval">
+
+    <solid android:color="@color/bg_rain" />
```

Comment:
you can set it programmatically 
`view.background = DrawableCellModel(
            drawable = shapeDrawable(shapeCircle()),
            tint = R.color.bg_rain,
        )`